### PR TITLE
Final

### PR DESCRIPTION
**Pull request by AckonSamuel**
In this pull request, I attempt to merge the changes we implemented so far on the 'Catalog of my things application' - a group capstone project where we created different classes to be used in the interaction of the console application. We also wrote tests and created an SQL file to describe the schema of the application.

Changes Made ♻️
 Setup Linters for the project
 Create the Item Class (with the properties and methods)
 Create the main.rb file as an entry point for our console application
 Create all required classes (as shown in the uml diagram) with their respective methods and properties
 Write unit tests for implemented classes
 Add respective options for each class for the console app
 Create a schema for the app using SQL
 Record group video
 Create the final PR (yes the one you're reading now 😄).
Group Video
Here is the link to the group video
https://drive.google.com/file/d/1xo1DM-BpKsGmBE35fATPzuiea0wvlRAm/view?usp=sharing

### CODE REVIEW

File: classes/author.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    @first_name = f_name
+
+    @last_name = l_name
+
+    @items = []
+  end
+
+  def add_item(new_item)
+    @items << (new_item)
+  end
```
Comment (Munsa1) [inline classes/author.rb:20]: - [ ] I am unable to add an author successfully, the program takes in input and upon enering the last value, loops back to the main menu, kindly fix this issue

File: classes/genre.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    @name = name
+    @items = []
+  end
+
+  def add_item(items)
+    case items
+    when Item
+      items.add_genre(self)
+      @items << items
+    else
```
Comment (Munsa1) [inline classes/genre.rb:18]: - [ ] I am unable to add a genre successfully, the program takes in input and upon enering the last value, loops back to the main menu, kindly fix this issue

File: classes/game.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+class Game < Item
+  attr_reader :multiplayer, :last_played
+
+  def initialize(multiplayer, last_played, publish_date)
+    super(publish_date)
+
+    @multiplayer = multiplayer
+    @last_played = last_played
+    move_to_archive
+  end
```
Comment (Munsa1) [inline classes/game.rb:13]: - [ ] I get the following error when adding a new game, kindly fix this issue.
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classes/game.rb (hunk 3 above)
Comment (AckonSamuel) [reply classes/game.rb:13]: Thank you. All issues have fixed

File: classes/author.rb (hunk 1 above)
Comment (AckonSamuel) [reply classes/author.rb:20]: Sally is a good girl

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey Team! 😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**